### ISSUE TITLE
SC2063 should not apply to `grep -F`?

### ISSUE THREAD

**Issue by kevgrig** [koalaman/shellcheck#824]
#### For bugs
- Rule Id: SC2063
- My shellcheck version: online
- [Y] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh
printf "foo" | grep -Fv '*'

```

#### Here's what shellcheck currently says:

```
Line 2:
printf "foo" | grep -Fv '*'
                        ^-- SC2063: Grep uses regex, but this looks like a glob.
```

#### Here's what I wanted or expected to see:

> No issues detected!

#### Details

> -F
>     Match using fixed strings. Treat each pattern specified as a string instead of a regular expression. If an input line contains any of the patterns as a contiguous sequence of bytes, the line shall be matched. A null string shall match every line.

http://pubs.opengroup.org/onlinepubs/9699919799/utilities/grep.html

**Comment by koalaman** [comment]
Fixed in 128d5d6. Thanks!

**Comment by dkubb** [comment]
Should this also apply when `--fixed-strings` is used?

**Comment by e-kwsm** [comment]
fixed by #1468
